### ISSUE TITLE
Go: logging sometimes doesn't.

### ISSUE THREAD

**Issue by kormat** [scionproto/scion#2001]
Sometimes the BR stops logging, in the middle of a line. E.g.:
```
2018-10-05 19:24:01.231422+0000 [DBUG] Forwarding revocati           
```
Logging never recovers without restart, and no go-routines are blocked trying to log, so most likely logging just goes to a `/dev/null`-style bitbucket. It's possible this coincides with the disk being full. My best guess is a failure happens in lumberjack.

**Comment by lukedirtwalker** [comment]
This also happenend in this build, in the PS1-110
https://buildkite.com/scionproto/scion/builds/530#821fea2d-9c0b-4f6d-8bb1-03cc159eb1e7

**Comment by scrye** [comment]
Regarding if it's a problem in lumberjack, but the Github project page has no issues mentioning this. That is not to say it's impossible, but it is still a data point.

I wrote some stress tests in order to reproduce the behavior; my tests created 20 goroutines and then sent 10.000.000 logging lines to a file in parallel. Everything worked ok. I tested both with and without log rotation, and was unable to reproduce the bug.

Additionally, we haven't seen it in quite a long time (> 6mths)

Closing with a won't fix.
